src/main/java/com/indeed/jiraactions/TSVSpecBuilder.java

```diff
@@ -99,4 +127,44 @@ private static String getAllLinksValue(final Action action) {
 
         return String.join(" ", values);
     }
+
+    private static long getTotalStatusTime(final String statusType, final Action action) {
```

Comment:
Instead of the second parameter being `Action`, consider having it be whatever the result of `action.getStatustimes()` is. That will make it significantly more testable. On a related note, this seems like a good thing to test ;)


This applies to `getTimeToLast` and `getTimeToFirst` as well.